app/models.py

```diff
@@ -516,12 +516,14 @@ def __init__(self,
                  type,
                  metadata_id,
                  privacy,
+                 release_date,
```

Comment:
For app.request.utils, the response of creating a request is private. Do we still have to add 'release_date=None'?
I have added the release_date to the parameter in test.lib.tools and ran the test suite and have confirmed that the usage is functioning.